app/controllers/donors_controller.rb

```diff
@@ -1,12 +1,23 @@
 class DonorsController < ApplicationController
+  include ActionView::Helpers::NumberHelper
   # GET donors/:id
   def show
     @donor = find_donor(params[:id])
   end
 
+  # GET /donors
+  def index
+    @donors = Donor.all
+    @total_amt = total_amt_donated
```

Comment:
Once we move the summation logic into `Donor`, we can remove this instance variable.